11/24/2021 8:57:00 PM: Warning: Action got this custom Exception :11/24/2021 8:57:00 PM: Info: Start method: Method_111/24/2021 8:57:00 PM: Warning: Action got this custom Exception :11/24/2021 8:57:00 PM: Info: Start method: Method_111/24/2021 8:57:01 PM: Error: Action failed by reason: System.Exception: I broke a logic
   at Modul2HW5.Services.Actions.Method_3() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Services\Actions.cs:line 31
   at Modul2HW5.Starter.Run() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Starter.cs:line 4711/24/2021 8:57:01 PM: Error: Action failed by reason: System.Exception: I broke a logic
   at Modul2HW5.Services.Actions.Method_3() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Services\Actions.cs:line 31
   at Modul2HW5.Starter.Run() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Starter.cs:line 4711/24/2021 8:57:01 PM: Warning: Action got this custom Exception :11/24/2021 8:57:01 PM: Info: Start method: Method_111/24/2021 8:57:02 PM: Warning: Action got this custom Exception :11/24/2021 8:57:02 PM: Error: Action failed by reason: System.Exception: I broke a logic
   at Modul2HW5.Services.Actions.Method_3() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Services\Actions.cs:line 31
   at Modul2HW5.Starter.Run() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Starter.cs:line 4711/24/2021 8:57:02 PM: Info: Start method: Method_111/24/2021 8:57:02 PM: Warning: Action got this custom Exception :11/24/2021 8:57:02 PM: Info: Start method: Method_111/24/2021 8:57:03 PM: Error: Action failed by reason: System.Exception: I broke a logic
   at Modul2HW5.Services.Actions.Method_3() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Services\Actions.cs:line 31
   at Modul2HW5.Starter.Run() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Starter.cs:line 4711/24/2021 8:57:03 PM: Warning: Action got this custom Exception :11/24/2021 8:57:03 PM: Info: Start method: Method_111/24/2021 8:57:03 PM: Warning: Action got this custom Exception :11/24/2021 8:57:03 PM: Warning: Action got this custom Exception :11/24/2021 8:57:03 PM: Info: Start method: Method_111/24/2021 8:57:03 PM: Info: Start method: Method_111/24/2021 8:57:03 PM: Info: Start method: Method_111/24/2021 8:57:03 PM: Warning: Action got this custom Exception :11/24/2021 8:57:03 PM: Error: Action failed by reason: System.Exception: I broke a logic
   at Modul2HW5.Services.Actions.Method_3() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Services\Actions.cs:line 31
   at Modul2HW5.Starter.Run() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Starter.cs:line 4711/24/2021 8:57:03 PM: Error: Action failed by reason: System.Exception: I broke a logic
   at Modul2HW5.Services.Actions.Method_3() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Services\Actions.cs:line 31
   at Modul2HW5.Starter.Run() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Starter.cs:line 4711/24/2021 8:57:03 PM: Info: Start method: Method_111/24/2021 8:57:03 PM: Warning: Action got this custom Exception :11/24/2021 8:57:03 PM: Warning: Action got this custom Exception :11/24/2021 8:57:03 PM: Error: Action failed by reason: System.Exception: I broke a logic
   at Modul2HW5.Services.Actions.Method_3() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Services\Actions.cs:line 31
   at Modul2HW5.Starter.Run() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Starter.cs:line 4711/24/2021 8:57:03 PM: Warning: Action got this custom Exception :11/24/2021 8:57:03 PM: Warning: Action got this custom Exception :11/24/2021 8:57:03 PM: Error: Action failed by reason: System.Exception: I broke a logic
   at Modul2HW5.Services.Actions.Method_3() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Services\Actions.cs:line 31
   at Modul2HW5.Starter.Run() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Starter.cs:line 4711/24/2021 8:57:03 PM: Error: Action failed by reason: System.Exception: I broke a logic
   at Modul2HW5.Services.Actions.Method_3() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Services\Actions.cs:line 31
   at Modul2HW5.Starter.Run() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Starter.cs:line 4711/24/2021 8:57:03 PM: Info: Start method: Method_111/24/2021 8:57:03 PM: Warning: Action got this custom Exception :11/24/2021 8:57:03 PM: Warning: Action got this custom Exception :11/24/2021 8:57:03 PM: Warning: Action got this custom Exception :11/24/2021 8:57:03 PM: Error: Action failed by reason: System.Exception: I broke a logic
   at Modul2HW5.Services.Actions.Method_3() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Services\Actions.cs:line 31
   at Modul2HW5.Starter.Run() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Starter.cs:line 4711/24/2021 8:57:03 PM: Error: Action failed by reason: System.Exception: I broke a logic
   at Modul2HW5.Services.Actions.Method_3() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Services\Actions.cs:line 31
   at Modul2HW5.Starter.Run() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Starter.cs:line 4711/24/2021 8:57:03 PM: Warning: Action got this custom Exception :11/24/2021 8:57:03 PM: Info: Start method: Method_111/24/2021 8:57:03 PM: Error: Action failed by reason: System.Exception: I broke a logic
   at Modul2HW5.Services.Actions.Method_3() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Services\Actions.cs:line 31
   at Modul2HW5.Starter.Run() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Starter.cs:line 4711/24/2021 8:57:04 PM: Error: Action failed by reason: System.Exception: I broke a logic
   at Modul2HW5.Services.Actions.Method_3() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Services\Actions.cs:line 31
   at Modul2HW5.Starter.Run() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Starter.cs:line 4711/24/2021 8:57:04 PM: Warning: Action got this custom Exception :11/24/2021 8:57:04 PM: Error: Action failed by reason: System.Exception: I broke a logic
   at Modul2HW5.Services.Actions.Method_3() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Services\Actions.cs:line 31
   at Modul2HW5.Starter.Run() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Starter.cs:line 4711/24/2021 8:57:04 PM: Error: Action failed by reason: System.Exception: I broke a logic
   at Modul2HW5.Services.Actions.Method_3() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Services\Actions.cs:line 31
   at Modul2HW5.Starter.Run() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Starter.cs:line 4711/24/2021 8:57:04 PM: Info: Start method: Method_111/24/2021 8:57:04 PM: Warning: Action got this custom Exception :11/24/2021 8:57:04 PM: Info: Start method: Method_111/24/2021 8:57:04 PM: Info: Start method: Method_111/24/2021 8:57:04 PM: Warning: Action got this custom Exception :11/24/2021 8:57:04 PM: Warning: Action got this custom Exception :11/24/2021 8:57:04 PM: Warning: Action got this custom Exception :11/24/2021 8:57:04 PM: Error: Action failed by reason: System.Exception: I broke a logic
   at Modul2HW5.Services.Actions.Method_3() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Services\Actions.cs:line 31
   at Modul2HW5.Starter.Run() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Starter.cs:line 4711/24/2021 8:57:04 PM: Warning: Action got this custom Exception :11/24/2021 8:57:04 PM: Error: Action failed by reason: System.Exception: I broke a logic
   at Modul2HW5.Services.Actions.Method_3() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Services\Actions.cs:line 31
   at Modul2HW5.Starter.Run() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Starter.cs:line 4711/24/2021 8:57:04 PM: Info: Start method: Method_111/24/2021 8:57:04 PM: Info: Start method: Method_111/24/2021 8:57:04 PM: Warning: Action got this custom Exception :11/24/2021 8:57:04 PM: Info: Start method: Method_111/24/2021 8:57:04 PM: Info: Start method: Method_111/24/2021 8:57:04 PM: Info: Start method: Method_111/24/2021 8:57:04 PM: Error: Action failed by reason: System.Exception: I broke a logic
   at Modul2HW5.Services.Actions.Method_3() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Services\Actions.cs:line 31
   at Modul2HW5.Starter.Run() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Starter.cs:line 4711/24/2021 8:57:04 PM: Error: Action failed by reason: System.Exception: I broke a logic
   at Modul2HW5.Services.Actions.Method_3() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Services\Actions.cs:line 31
   at Modul2HW5.Starter.Run() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Starter.cs:line 4711/24/2021 8:57:04 PM: Info: Start method: Method_111/24/2021 8:57:04 PM: Info: Start method: Method_111/24/2021 8:57:04 PM: Warning: Action got this custom Exception :11/24/2021 8:57:04 PM: Info: Start method: Method_111/24/2021 8:57:04 PM: Error: Action failed by reason: System.Exception: I broke a logic
   at Modul2HW5.Services.Actions.Method_3() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Services\Actions.cs:line 31
   at Modul2HW5.Starter.Run() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Starter.cs:line 4711/24/2021 8:57:04 PM: Info: Start method: Method_111/24/2021 8:57:04 PM: Error: Action failed by reason: System.Exception: I broke a logic
   at Modul2HW5.Services.Actions.Method_3() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Services\Actions.cs:line 31
   at Modul2HW5.Starter.Run() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Starter.cs:line 4711/24/2021 8:57:04 PM: Warning: Action got this custom Exception :11/24/2021 8:57:04 PM: Error: Action failed by reason: System.Exception: I broke a logic
   at Modul2HW5.Services.Actions.Method_3() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Services\Actions.cs:line 31
   at Modul2HW5.Starter.Run() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Starter.cs:line 4711/24/2021 8:57:04 PM: Info: Start method: Method_111/24/2021 8:57:04 PM: Info: Start method: Method_111/24/2021 8:57:04 PM: Warning: Action got this custom Exception :11/24/2021 8:57:04 PM: Info: Start method: Method_111/24/2021 8:57:04 PM: Info: Start method: Method_111/24/2021 8:57:04 PM: Error: Action failed by reason: System.Exception: I broke a logic
   at Modul2HW5.Services.Actions.Method_3() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Services\Actions.cs:line 31
   at Modul2HW5.Starter.Run() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Starter.cs:line 4711/24/2021 8:57:04 PM: Info: Start method: Method_111/24/2021 8:57:04 PM: Error: Action failed by reason: System.Exception: I broke a logic
   at Modul2HW5.Services.Actions.Method_3() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Services\Actions.cs:line 31
   at Modul2HW5.Starter.Run() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Starter.cs:line 4711/24/2021 8:57:04 PM: Error: Action failed by reason: System.Exception: I broke a logic
   at Modul2HW5.Services.Actions.Method_3() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Services\Actions.cs:line 31
   at Modul2HW5.Starter.Run() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Starter.cs:line 4711/24/2021 8:57:04 PM: Info: Start method: Method_111/24/2021 8:57:04 PM: Error: Action failed by reason: System.Exception: I broke a logic
   at Modul2HW5.Services.Actions.Method_3() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Services\Actions.cs:line 31
   at Modul2HW5.Starter.Run() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Starter.cs:line 4711/24/2021 8:57:04 PM: Error: Action failed by reason: System.Exception: I broke a logic
   at Modul2HW5.Services.Actions.Method_3() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Services\Actions.cs:line 31
   at Modul2HW5.Starter.Run() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Starter.cs:line 4711/24/2021 8:57:04 PM: Info: Start method: Method_111/24/2021 8:57:04 PM: Info: Start method: Method_111/24/2021 8:57:04 PM: Error: Action failed by reason: System.Exception: I broke a logic
   at Modul2HW5.Services.Actions.Method_3() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Services\Actions.cs:line 31
   at Modul2HW5.Starter.Run() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Starter.cs:line 4711/24/2021 8:57:04 PM: Error: Action failed by reason: System.Exception: I broke a logic
   at Modul2HW5.Services.Actions.Method_3() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Services\Actions.cs:line 31
   at Modul2HW5.Starter.Run() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Starter.cs:line 4711/24/2021 8:57:04 PM: Error: Action failed by reason: System.Exception: I broke a logic
   at Modul2HW5.Services.Actions.Method_3() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Services\Actions.cs:line 31
   at Modul2HW5.Starter.Run() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Starter.cs:line 4711/24/2021 8:57:04 PM: Warning: Action got this custom Exception :11/24/2021 8:57:04 PM: Info: Start method: Method_111/24/2021 8:57:04 PM: Warning: Action got this custom Exception :11/24/2021 8:57:04 PM: Error: Action failed by reason: System.Exception: I broke a logic
   at Modul2HW5.Services.Actions.Method_3() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Services\Actions.cs:line 31
   at Modul2HW5.Starter.Run() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Starter.cs:line 4711/24/2021 8:57:04 PM: Info: Start method: Method_111/24/2021 8:57:04 PM: Error: Action failed by reason: System.Exception: I broke a logic
   at Modul2HW5.Services.Actions.Method_3() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Services\Actions.cs:line 31
   at Modul2HW5.Starter.Run() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Starter.cs:line 4711/24/2021 8:57:04 PM: Error: Action failed by reason: System.Exception: I broke a logic
   at Modul2HW5.Services.Actions.Method_3() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Services\Actions.cs:line 31
   at Modul2HW5.Starter.Run() in C:\Users\admin\source\repos\Sofia-Rutskaya\Modul2HW5\Modul2HW5\Modul2HW5\Modul2HW5\Starter.cs:line 4711/24/2021 8:57:04 PM: Info: Start method: Method_111/24/2021 8:57:04 PM: Info: Start method: Method_111/24/2021 8:57:04 PM: Warning: Action got this custom Exception :11/24/2021 8:57:04 PM: Warning: Action got this custom Exception :